clients/test/plugins/plugin-test/src/lib/cmds/metadata.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const metadataWithNameOnly = {
+  metadata: {
+    name: 'this is the name part'
+  }
+}
```

Comment:
Refactored the minimal metadata requirement into its own file, so that the tests could import directly rather than having their own version.